package.json

```diff
@@ -31,7 +31,8 @@
     "storybook": "nps storybook.server",
     "build-storybook": "build-storybook",
     "chromatic": "chromatic",
-    "chromatic-ci": "build-storybook && chromatic --ci --exit-zero-on-changes"
+    "chromatic-ci": "build-storybook && chromatic --ci --exit-zero-on-changes",
+    "test": "jest --config=jest-config.js"
```

Comment:
@ryan-roemer Right now, we are able to run jest tests using the `jest` command. However, we need to manually rebuild modules using `nps build-package-libs` every time there are code changes in a package. Re-building before running each test is what has been causing the current test suite to be so slow. You added the `TEST_MODULE` flag last year that will build one package in isolation, but this doesn't work for automatically re-building changed files if we are running jest in watch mode. Do you know of any tooling that could handle this better?

Reply:
So, there are two things really:

1. Build **all** lib and test code when running tests.
2. Build only test code when running tests. For lib code that means running a separate `watch` to rebuild or one-off build.

I think No. 2 is the way to go.

And for Jest `TEST_MODULE` doesn't really apply. And for Karma, it has very little actual speed impact. The thing that _does_ now massively speed up Karma in my d3 branch is that I only use already build lib code and remove all transpiling and inclusion of source code.